src/config.ts

```diff
@@ -1,7 +1,8 @@
 const {
     NATS_URL = "",
     ELASTICSEARCH_URI = "",
-    ELASTICSEARCH_INDEX_NAME = ""
+    ELASTICSEARCH_INDEX_NAME = "",
+    ELASTICSEARCH_INVENTORY_INDEX_NAME = ""
```

Comment:
yes, we need a new index for inventory items.
We should use the `ELASTICSEARCH_INDEX_NAME` and append the `-inventory` name so all related indexes have a common stem.
When this PR is merged in, you should be able to just reuse `INDEXES.INVENTORY`: https://github.com/jwa-lab/item-store/pull/8/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R28